src/ipc/Queue.cc

```diff
@@ -238,7 +217,7 @@ Ipc::FewToFewBiQueue::FewToFewBiQueue(const String &id, const Group aLocalGroup,
     theLocalGroup(aLocalGroup)
 {
     Must(queues->theCapacity == metadata->theGroupASize * metadata->theGroupBSize * 2);
-    Must(readers->theCapacity == metadata->theGroupASize + metadata->theGroupBSize);
+    Must(readers->theCapacity == size_t(metadata->theGroupASize) + size_t(metadata->theGroupBSize));
```

Comment:
On the TODO, but at the current stage of the refactor that would introduce even more casting. It needs to wait until more FooQueue classes have also been cleaned.

Reply:
Please clean all these classes -- they work together. Cleaning a few (without proper testing) instead of the whole set is worse than leaving them as is (in a bad but tested state).